syncstorage-spanner/src/metadata.rs

```diff
@@ -93,11 +95,12 @@ mod tests {
         MetadataBuilder, LEADER_AWARE_KEY, METRICS_KEY, PREFIX_KEY, ROUTING_KEY, USER_AGENT,
     };
 
-    pub const DB: &str = "/projects/sync/instances/test/databases/sync1";
-    pub const SESSION: &str = "/projects/sync/instances/test/databases/sync1/sessions/f00B4r_quuX";
+    // Resource paths should not start with a "/"
+    pub const DB: &str = "projects/sync/instances/test/databases/sync1";
+    pub const SESSION: &str = "projects/sync/instances/test/databases/sync1/sessions/f00B4r_quuX";
```

Comment:
FYI I double checked that we're in fact sending such values without a leading slash, this was just a mistake in the test's data.

Reply:
Yep, just wanted to make sure that the test values reflected the correct value, as well as the guidance was captured in the code.

Reply:
:100: